gradle.properties

```diff
@@ -2,7 +2,7 @@ org.gradle.daemon=false
 org.gradle.parallel=false
 org.gradle.jvmargs=-Xms512m -Xmx512m
 scalaVersion=2.13.6
```

Comment:
Should we bump the Scala version to match the one used in Kafka 3.1 (Scala 2.13.8)?

Reply:
I guess we could. Does it matter though?